site-src/index.md

```diff
@@ -94,8 +94,31 @@ This flexibility allows the API to adapt to vastly different
 organizational models and implementations while remaining a portable and
 standard API.
 
+## What's the difference between Gateway API and an API Gateway?
 
-## Who is working on Gateway?
+A Gateway is a networking component that allows data to transfer from one
+network to another network.
+
+An API Gateway is a Gateway that deals more specificially with backend services
+and routes client reqeusts to those backend services. Those services expose an
+API such as HTTP, REST or some other protocol on top of TCP or UDP.
+
+Gateway API is an interface, or set of resources, that model service networking
+in Kubernetes. One of the main resources is a `Gateway`. If you implement
+Gateway API, you are a Gateway provider. A `Gateway` resource declares the
```

Comment:
I think it's a little bit confusing to say "If you implement Gateway API, you are a Gateway provider", how about "As a Gateway Provider, you can implement the Gateway API to model Kubernetes service networking in an expressive, extensible, and role-oriented way." ?

Reply:
Thanks.
All great suggestions, which have now been incorporated